SKILL.tsv

```diff
@@ -4994,16 +4994,16 @@ SKILL_20150730_004993	Make Shogos molded based on the card inserted in Necronomi
 SKILL_20150730_004994	Your companion bites and holds onto a target. Decreases the physical defense of the bitten target. You receive a critical chance bonus against the target.
 SKILL_20150730_004995	$Physical Defense: -#{CaptionRatio}# {nl}Additional Critical Chance: 30%{nl}Duration: #{CaptionTime}# seconds
 SKILL_20150803_004996	$Quick Cast
-SKILL_20150803_004997	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Sit down to increase focus. ATK, speed and range increases.
+SKILL_20150803_004997	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Kneel to increase focus. Physical Attack, Attack Speed, and range increase.
```

Comment:
Physical Attack, Attack Speed, and range increase. -> Increases (your) attack range and attack speed.